libsolidity/ast/AST.h

```diff
@@ -648,11 +649,15 @@ class InheritanceSpecifier: public ASTNode
  * 2. `using LibraryName for *` attaches to all types.
  * 3. `using {f1, f2, ..., fn} for T` attaches the functions `f1`, `f2`, ...,
  *     `fn`, respectively to `T`.
+ * 4. `using {f1 as op1, f2 as op2, ..., fn as opn} for T` bounds operator `opn` to function `fn` for type `T`.
```

Comment:
```suggestion
 * 4. `using {f1 as op1, f2 as op2, ..., fn as opn} for T` binds operator `opn` to function `fn` for type `T`.
```
Or maybe "attaches"? Or is the function bound to the operator or the operator bound to the function?

I think we should establish a common vocabulary for this. For example in the docs you said `a list of assignments of functions to operators`. Of the three options I think that "a function bound to an operator" sounds natural and is consistent with "a function bound to a type" which is how we call attached functions already (they even have a `bound()` method).

Maybe we should also rename `FunctionType::bound()` to something like `boundToType()` because bound can now refer to two things?

Reply:
"A function bound to an operator" sounds good. I updated code and documentation. Also, I renamed `FunctionType::bound()` as suggested.

Reply:
There were a few remaining places that I now fixed. Also, extracted this refactor into #13741.